packages/core/src/checkout-buttons/strategies/amazon-pay-v2/amazon-pay-v2-button-options.ts

```diff
@@ -1,6 +1,6 @@
-import { AmazonPayV2ButtonParams } from '../../../payment/strategies/amazon-pay-v2';
+import { AmazonPayV2ButtonParameters } from '../../../payment/strategies/amazon-pay-v2';
 
 /**
  * The required config to render the AmazonPayV2 buttton.
  */
-export type AmazonPayV2ButtonInitializeOptions = AmazonPayV2ButtonParams;
+export type AmazonPayV2ButtonInitializeOptions = AmazonPayV2ButtonParameters;
```

Comment:
```suggestion
export type AmazonPayV2ButtonInitializeOptions = AmazonPayV2ButtonParams;
```

Reply:
@animesh1987 Are you suggesting a name change or a type change? Because `AmazonPayV2ButtonParameters` is the right type, otherwise I'll be omitting the new params. 😬